### ISSUE TITLE
Dryopteris borreri morphotype robusta sensu Trewren (2014) BPS

### ISSUE THREAD

**Issue by GeoffToone** [BSBI/mapmate-taxon-requests#25]
I don't know the current situation with Stace 3 but have a sample recently determined as this.

**Comment by DavidBroughton** [comment]
I agree would be useful to facilitate segregation of records given this is part of an apomictic group. Sell & Murrell Volume 1 include as D. robusta (Oberh. & Tavel ex Fraser-Jenk.) P.D. Sell   Robust Scaly Male-fern, and also note the synonym D. borreri var. robusta Oberh. & Tavel. In DDB as morph. robusta https://database.bsbi.org/object.php?entityid=2cd4p9h.6y1s2n&class=TaxonInstance   
No specific preference on how it enters the list as long as there is a name to use. for existing and future records
